app/src/visionglass/res/values/themes.xml

```diff
@@ -15,6 +15,156 @@
         <item name="backgroundColor">@color/midnight</item>
     </style>
 
+    <style name="ShapeAppearanceOverlay.App.Button.Rounded" parent="ShapeAppearance.MaterialComponents.MediumComponent">
+        <item name="cornerFamily">rounded</item>
+        <item name="cornerSize">32dp</item>
+    </style>
+
+    <style name="ThemeOverlay.App.Slider" parent="">
+        <item name="colorPrimary">@color/white</item>
+        <item name="colorOnPrimary">@color/white</item>
+    </style>
+
+    <style name="BaseToolbarButton" parent="Widget.MaterialComponents.Button">
+        <item name="android:layout_width">0dp</item>
+        <item name="android:layout_height">?attr/actionBarSize</item>
+        <item name="android:backgroundTint">@color/bg_button_checkable</item>
+        <item name="android:scaleType">fitCenter</item>
+        <item name="android:layout_margin">8dp</item>
+        <item name="iconGravity">textStart</item>
+        <item name="iconPadding">0dp</item>
+        <item name="iconTint">@color/white</item>
+        <item name="shapeAppearanceOverlay">@style/ShapeAppearanceOverlay.App.Button.Rounded</item>"
+    </style>
+
+    <style name="BaseFooterButton" parent="Widget.MaterialComponents.Button">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">?attr/actionBarSize</item>
+        <item name="android:scaleType">fitCenter</item>
+        <item name="android:backgroundTint">@color/asphalt</item>
```

Comment:
Can we use the same color here as the other buttons in the UI?

    <item name="android:backgroundTint">@color/bg_button_checkable</item>

Reply:
I'm not sure, tbh, I'm also working on a Tray for the same buttons we have in the Wolvic UI and I had plans to use different color patterns to differentiate.